CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+<a name="0.8.0"></a>
+# 0.8.0 (2016-11-25)
+
+### Features
+
+* Angular 2.2.3
+* Dependencies updated
+
+### Bug Fixes
+
+
+### How to update
+
+* Pull sources from git, merge accordingly
+* remove `node_modules`
+* run `npm install`
+* Remove all directives & pipes from components and move them to the ngModule declarations
```

Comment:
@nnixaa I tested it on my project and everything worked fine without moving anything.